.travis.yml

```diff
@@ -1,6 +0,0 @@
-language: elixir
-otp_release:
-  - 20.0
-elixir:
-  - 1.6
-sudo: false
```

Comment:
do we need to update README.md too? 👀

Reply:
I was just looking at updating the badge.

Test badge...
![testing out badges](https://github.com/therealreal/prismic-elixir/actions/workflows/ci.yml/badge.svg)